src/patternfly/components/AboutModalBox/about-modal-box.scss

```diff
@@ -10,79 +10,72 @@
+  --pf-c-about-modal-box--lg--MinHeight: calc(var(--pf-c-about-modal-box__brand--PaddingTop) + var(--pf-c-about-modal-box__brand--PaddingBottom) + var(--pf-c-about-modal-box__header--PaddingTop) + var(--pf-c-about-modal-box__header--PaddingBottom) + var(--pf-c-about-modal-box__content--PaddingTop) + var(--pf-c-about-modal-box__content--PaddingBottom) + var(--pf-c-about-modal-box__brand-image--Height) + (1rem * 6 * var(--pf-global--LineHeight--md)));
 
   // MaxHeight ensures that the modal will not go off the screen and instead the content will scroll
-  --pf-c-about-modal-box--MaxHeight: calc(100vh - var(--pf-global--spacer--xl));
+  --pf-c-about-modal-box--lg--MaxHeight: calc(100vh - var(--pf-global--spacer--xl));
 
   // Margin left and right make sure there is some space around the modal at the largest breakpoint
-  --pf-c-about-modal-box--MarginLeft: var(--pf-global--spacer--3xl);
-  --pf-c-about-modal-box--MarginRight: var(--pf-global--spacer--3xl);
+  --pf-c-about-modal-box--lg--MarginX: var(--pf-global--spacer--3xl);
 
   // Shared padding values allow us to change common padding to all components once
   // Some modal components have more specific padding values
-  --pf-c-about-modal-box--PaddingTop: var(--pf-global--spacer--xl);
-  --pf-c-about-modal-box--PaddingRight: var(--pf-global--spacer--xl);
-  --pf-c-about-modal-box--PaddingBottom: var(--pf-global--spacer--xl);
-  --pf-c-about-modal-box--PaddingLeft: var(--pf-global--spacer--xl);
+  --pf-c-about-modal-box--Padding: var(--pf-global--spacer--xl);
+  --pf-c-about-modal-box--sm--Padding: var(--pf-global--spacer--3xl);
 
   @media screen and (min-width: $pf-global--breakpoint--sm) {
-    --pf-c-about-modal-box--PaddingTop: var(--pf-global--spacer--3xl);
-    --pf-c-about-modal-box--PaddingRight: var(--pf-global--spacer--3xl);
-    --pf-c-about-modal-box--PaddingBottom: var(--pf-global--spacer--3xl);
-    --pf-c-about-modal-box--PaddingLeft: var(--pf-global--spacer--3xl);
+    --pf-c-about-modal-box--Padding: var(--pf-c-about-modal-box--sm--Padding);
   }
 
   // Element variables
 
   // Brand
   // Set default padding top, right, left to shared values
   --pf-c-about-modal-box__brand--PaddingTop: var(--pf-global--spacer--2xl);
-  --pf-c-about-modal-box__brand--PaddingRight: var(--pf-c-about-modal-box--PaddingRight);
-  --pf-c-about-modal-box__brand--PaddingBottom: var(--pf-c-about-modal-box--PaddingBottom);
-  --pf-c-about-modal-box__brand--PaddingLeft: var(--pf-c-about-modal-box--PaddingLeft);
+  --pf-c-about-modal-box__brand--PaddingRight: var(--pf-c-about-modal-box--Padding);
+  --pf-c-about-modal-box__brand--PaddingBottom: var(--pf-c-about-modal-box--Padding);
+  --pf-c-about-modal-box__brand--PaddingLeft: var(--pf-c-about-modal-box--Padding);
 
   // Close
   // Set default padding top, right, bottom to shared values
```

Comment:
IMO `--pf-c-about-modal-box__close--PaddingTop` being assigned to `var(--pf-c-about-modal-box__brand--PaddingTop)` is confusing . I understand that its good to reuse variables but I wonder what happens if the brand PaddingTop changes, and we don't want the box_close PaddingTop to change? But thats just my thought!